### PR TITLE
Add video helpers

### PR DESCRIPTION
**Pull request by manuquentin**

### CODE REVIEW

File: src/web-rtc-client.js [hunk 1]
```diff
@@ -722,6 +743,10 @@ export default class WebRTCClient extends Emitter {
 
   _createWebRTCConfiguration() {
     const config: Object = {
+      // @TEMP + is overwritten in mobile.
```
Comment (sboily) [inline src/web-rtc-client.js:746]: Why this comment here?

Comment (manuquentin) [reply src/web-rtc-client.js:746]: @bogue ?

File: src/web-rtc-client.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ const events = [
   'transactionDestroyed',
   'notify',
   'outOfDialogReferRequested',
-  'message',
+  'message', // i believe this is overwritten by its namesake in transportEvents
```
Comment (sboily) [inline src/web-rtc-client.js:39]: Same why this comment?

File: src/web-rtc-client.js [hunk 3]
```diff
@@ -722,6 +743,10 @@ export default class WebRTCClient extends Emitter {
 
   _createWebRTCConfiguration() {
     const config: Object = {
+      // @TEMP + is overwritten in mobile.
+      registerOptions: {
+        expires: 6000, // default 600
```
Comment (sboily) [inline src/web-rtc-client.js:748]: Can you use 3600 by default please.

**Comment by manuquentin** [comment]
As seen with @sboily we can merge it to continue other development

**Pull request merged**